tikv-configuration-file.md

```diff
@@ -737,23 +737,23 @@ Configuration items related to RocksDB
 
 ### `rate-bytes-per-sec`
 
-+ The maximum rate permitted by Rate Limiter
-+ Default value: `0`
++ The maximum rate permitted by RocksDB compaction rate limiter
```

Comment:
RocksDB' compaction rate limiter? I found no result of "RocksDB compaction rate limiter" in Google.

Reply:
The added "compaction" is mainly to differentiate from a new rate limiter we are about to introduce next month. It's also technically correct to use because RocksDB limiter doesn't regulate foreground IO.